meta-balena-common/recipes-containers/balena/balena.bb

```diff
@@ -15,17 +15,15 @@ inherit goarch
 inherit pkgconfig
 inherit useradd
 
-BALENA_VERSION = "20.10.17"
-BALENA_BRANCH= "master"
+BALENA_VERSION = "v20.10.17"
 
-SRCREV = "13db38c82bdb056f013f5497b0662ad34ffb98f7"
```

Comment:
@lmbarros Am I correct in my assumption that merges to balena-engine will always have a unique tag?

Maybe I need a rundown on engine versioning, but it seems to be that `{major}.{minor}` comes from upstream, and everything we do only increments `{patch}` ? Is that a safe assumption or should we stay with the commit hash?